assembl/static2/js/app/components/debate/brightMirror/fictionCommentList.jsx

```diff
@@ -45,7 +44,7 @@ const FictionCommentList = ({
       InnerComponent={FictionComment}
       InnerComponentFolded={FoldedPost}
       noRowsRenderer={noRowsRenderer}
-      SeparatorComponent={InfiniteSeparator}
+      SeparatorComponent={() => null}
```

Comment:
Curious: why do you need to put this function here ?

Reply:
@MilanBarande in our case we don't need a separator between debate comment posts for BM. We thus have an empty component since it is a required props for Tree (I couldn't have turned that props into optional but that would insert regression that I won't be aware of since there is no unit tests related to Tree.jsx). Hope that answer your question.